.gitignore

```
@@ -0,0 +1,5 @@
+poetry.lock
+__pycache__/bio_files_processor.cpython-312.pyc
+__pycache__/das_biotools.cpython-312.pyc
+__pycache__/test_das_biotools.cpython-312-pytest-8.2.0.pyc
+__pycache__/custom_random_forest.cpython-312.pyc
```

гитигнор тоже можно добавить в гитигнор
```suggestion
poetry.lock
__pycache__/*
.gitignore
```